Package.swift

```diff
@@ -1,15 +1,39 @@
 // swift-tools-version: 5.8
 
 import PackageDescription
+import CompilerPluginSupport
+
+// Availability Macros
+let availabilityTags = [_Availability("AsyncAlgorithms")]
+let versionNumbers = ["1.0"]
+
+// Availability Macro Utilities
+enum _OSAvailability: String {
+  // This should match the package's deployment target
+  case alwaysAvailable = "macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0"
```

Comment:
naming nit: this should probably be more so named `initialIntroduction` and not the "alwaysAvailable" since this is not something that has always been around.